chain/network/src/routing.rs

```diff
@@ -382,6 +384,29 @@ impl Default for EdgeVerifierHelper {
     }
 }
 
+#[cfg(feature = "adversarial")]
+pub struct TestHelper {
```

Comment:
This name feels too general. `NetworkTestHelper` might be better

Reply:
Thanks, that seems like a good idea to rename it. Done